lib/vmq_elixir_plugin.ex

```diff
@@ -22,17 +20,7 @@ defmodule VmqElixirPlugin do
   Application start function.
   """
   def start(_type, _args) do
-    import Supervisor.Spec, warn: false
-
-    children = [
-      # Define workers and child supervisors to be supervised
-      # worker(VmqElixirPlugin.Worker, [arg1, arg2, arg3])
-    ]
-
-    # See http://elixir-lang.org/docs/stable/elixir/Supervisor.html
-    # for other strategies and supported options
-    opts = [strategy: :one_for_one, name: VmqElixirPlugin.Supervisor]
-    Supervisor.start_link(children, opts)
+    :ok
```

Comment:
I am a bit confused now, I think that the Application behaviour expect a `start/2` to be defined and follow the spec: `start(app, start_type) :: :ok | {:error, term}`, not `{:ok, pid}`